### PR TITLE
on upgrade, don't roll back clusters that are on later versions

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

File: pkg/install/upgrade.go [hunk 1]
```diff
@@ -14,12 +14,32 @@ import (
...
+			i.log.Printf("not upgrading: cvo channel is %s", cv.Spec.Channel)
+			return nil
+		}
+
+		desired, err := version.ParseVersion(cv.Status.Desired.Version)
+		if err != nil {
+			return err
+		}
+
+		if !desired.Lt(vsn) {
```
Comment (kwoodson) [inline pkg/install/upgrade.go:38]: Just a question here about the ability to downgrade.  Is there a scenario where we would want to downgrade?

Comment (jim-minter) [reply pkg/install/upgrade.go:38]: It can be done manually if needed.

**Comment by asalkeld** [comment]
CI is getting "go: cannot find GOROOT directory: /usr/local/go1.13"

**Comment by mjudeikis** [comment]
Flake in CI: https://github.com/Azure/ARO-RP/issues/697

**Pull request merged**